region/tourian/main.json

```diff
@@ -3970,6 +3970,40 @@
                     "Run into the pirates with speedbooster to save health over shinesparking.",
                     "A speedy jump can reach the top platform."
                   ]
+                },
+                {
+                  "name": "G-Mode Morph PB Pirate Kill",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "h_canArtificialMorphSpringBallBombJump"
+                    ]},
+                    {"ammo": { "type": "PowerBomb", "count": 5 }}
```

Comment:
With canHitBox, you can get through with 1PB on first pirate and 2 for the three pack.